components/movementsensor/adxl345/interrupts.go

```diff
@@ -0,0 +1,80 @@
+package adxl345
+
+// addresses relevant to interrupts.
+const (
+	// contains an unsigned time value representing the maximum time that an event must be above the
```

Comment:
nit: Remove the first "time" and replace the second "time" with "duration". "Time" sounds like "timestamp," which is misleading here.

Similar comment on line 14